packages/nx/src/project-graph/utils/retrieve-workspace-files.ts

```diff
@@ -84,6 +85,8 @@ export async function retrieveWorkspaceFiles(
     'get-workspace-files:end'
   );
 
+  unregisterPluginTSTranspiler();
```

Comment:
Would the transpiler not just get re-registered when creating the dependencies? It seems like this should be done at the end of `createProjectGraphAsync` or similar as well.